docs/user-guide/cli-configuringcli-evfile.md

```diff
@@ -0,0 +1,62 @@
+# Configuring an environment variables file
+
+If it is not possible to configure your own system environment variables, create a special configuration file to set these variables for Zowe CLI commands.
+
+Although not common, there are cases where users do not have the ability to configure their own system environment variables. This can happen when users are working on hosted integrated development environments (IDEs), or in a highly locked down environment.
+
+When working under these kinds of restrictions, you can set environment variables that apply to CLI commands. To do this, create a `zowe.env.json` file storing key-value pairs that specify your configurations.
+
+**Note:** Use a `zowe.env.json` file *only* when it is not possible to set your own system environment variables. If you are able to configure environment variables in your system, continue to do so.
+
+## How `zowe.env.json` works
+
+When a Zowe command is issued, the command initializes the Imperative CLI Framework so that it loads all the utilities that allow the command to function. Imperative reads the `zowe.env.json` configuration file and sets the environment variables before any loggers or Zowe CLI finish their own initialization.
+
+The `zowe.env.json` environment variables are set for only the duration of a Zowe CLI command.
+
+If an existing environment variable is set in your system and the variable is also in `zowe.env.json`, the values in `zowe.env.json` overwrite it.
+
+`zowe.env.json` can be used to set any environment variable. This allows setting environment variables to change the default behavior of Node.JS and ibm_db, in addition to all of the Zowe environment variables.
+
+## Creating the configuration file
+
+Create a dedicated JSON file to store settings for environment variables.
+
+Follow these steps:
+
+1. In your Files Explorer, go to the home directory (%HOMEPATH% for Windows, $HOME for Linux and Mac) or the path set in the ZOWE_CLI_HOME environment variable.
+
+2. Create a JSON file titled `zowe.env.json`.
+
+    **Note:** Zowe CLI checks for this file when a command is issued. The file location must remain static and unchanged.
```

Comment:
Could you help me understand why we need to emphasize that the file location shouldn't be changed?

I guess the only way they can change the location is by changing the ZOWE_CLI_HOME on their environment?

Reply:
we discussed this offline. updated based on this.